WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -124,6 +124,8 @@ def process_nav_items(nav_items)
 
   def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
-    date_range(from_date, to_date, options)
+    if from_date && to_date
```

Comment:
Great, while you're at it, could you audit the other callers and see if they're checking as well?